frontend/src/components/Sidebar/useSidebarItems.ts

```diff
@@ -5,7 +5,17 @@ import helpers from '../../helpers';
 import { useCluster } from '../../lib/k8s';
 import { createRouteURL } from '../../lib/router';
 import { useTypedSelector } from '../../redux/reducers/reducers';
-import { DefaultSidebars, SidebarEntry, SidebarItemProps } from '.';
+import { DefaultSidebars, SidebarItemProps } from '.';
+
+/** Iterates over every entry in the list, including children */
+const forEachEntry = (items: SidebarItemProps[], cb: (item: SidebarItemProps) => void) => {
+  items.forEach(it => {
+    cb(it);
+    if (it.subList) {
+      forEachEntry(it.subList, cb);
+    }
```

Comment:
Hopefully we don't stack overflow.

Reply:
I'd be very impressed if someone constructs such a deep sidebar tree, especially since we only really support 2 levels